cfgov/v1/management/commands/import_translation_links.py

```diff
@@ -0,0 +1,120 @@
+# Import page translation links from a CSV file with format:
+#
+#    translated_page_relative_url,translated_page_english_url
```

Comment:
should the URL provided in translated_page_english_url be relative or absolute?

Reply:
Both should be relative! I'll update the script.